query_execution/tests/QueryManager_unittest.cpp

```diff
@@ -28,7 +29,7 @@
 #include "query_execution/QueryExecutionMessages.pb.h"
 #include "query_execution/QueryExecutionState.hpp"
 #include "query_execution/QueryExecutionTypedefs.hpp"
-#include "query_execution/QueryManager.hpp"
+#include "query_execution/QueryManagerSingleNode.hpp"
```

Comment:
As we are testing only the QueryManagerSingleNode related functionality, we can rename the file to ``QueryManagerSingleNode_unittest.cpp``